src/DotNet.Status.Web/Program.cs

```diff
@@ -27,10 +27,10 @@ public static void Main(string[] args)
                 .ConfigureAppConfiguration((host, config) =>
                 {
                     config
-                        .AddCommandLine(args)
+                        .AddDefaultJsonConfiguration(host.HostingEnvironment)
```

Comment:
Reviewers: Especially note this. The order seemed backwards as I was trying to let user-secrets hold the testing secrets I wanted to use but it be overwritten by whatever was in the configuration files. This now matches aspnet's [default behavior](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-5.0).